src/helpers/did.rs

```diff
@@ -0,0 +1,522 @@
+use std::str::FromStr;
+
+use crate::api::{VadeEvan, VadeEvanError};
+use crate::helpers::datatypes::{
+    AddPublicKeys,
+    AddServices,
+    CreateDidPayload,
+    DIDOperationType,
+    DidUpdatePayload,
+    Patch,
+    PublicKeyJWK,
+    PublicKeyModel,
+    PublicKeyPurpose,
+    RemovePublicKeys,
+    RemoveServices,
+    Service,
+    EVAN_METHOD,
+    TYPE_BBS_KEY,
+    TYPE_JSONWEB_KEY,
+    TYPE_SIDETREE_OPTIONS,
+};
+use base64::{decode_config, encode_config, URL_SAFE_NO_PAD};
+use uuid::Uuid;
+
+pub struct DID<'a> {
+    vade_evan: &'a mut VadeEvan,
+}
+
+impl<'a> DID<'a> {
+    pub fn new(vade_evan: &'a mut VadeEvan) -> Result<DID, VadeEvanError> {
+        Ok(DID { vade_evan })
+    }
+
+    pub async fn create(
+        self,
+        bbs_key: Option<&str>,
```

Comment:
Can you rename this to `bbs_public_key`? This then also requires updates in `vade_evan_api` and places where it's consumed.

Reply:
done